src/main/java/frc/robot/RobotContainer.java

```diff
@@ -65,6 +66,16 @@ public class RobotContainer {
   public RobotContainer() {
     configureButtonBindings();
 
+    m_armSubsystem.setDefaultCommand(
```

Comment:
Operator joystick controls should go under driver controls for formatting purposes